<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>66</width>
    <height>22</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Temp</string>
  </property>
  <property name="locale">
   <locale language="English" country="Sweden"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>3</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Courier 10 Pitch</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
  </action>
  <action name="action_Update">
   <property name="text">
    <string>&amp;Update</string>
   </property>
   <property name="toolTip">
    <string>Update temp</string>
   </property>
  </action>
  <action name="action_Change">
   <property name="text">
    <string>&amp;Reread cfg</string>
   </property>
   <property name="toolTip">
    <string>Reread onfig file</string>
   </property>
  </action>
  <action name="actionSet_host">
   <property name="text">
    <string>Set &amp;host</string>
   </property>
   <property name="toolTip">
    <string>Set hosts for onitoring and temp fetching</string>
   </property>
  </action>
  <action name="actionSavecfg">
   <property name="text">
    <string>&amp;Save cfg</string>
   </property>
   <property name="toolTip">
    <string>Save active config</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
